openhtf/io/output/print_ToConsole.py

```diff
@@ -0,0 +1,93 @@
+"""Module to display test summary on console."""
+import os
+import sys
+from openhtf import util
+from openhtf.io import test_record as tr
+from openhtf.util import measurements as mea
+
+class PrintToConsole():
+  """Print test results with failur info on console. """
+
+  def __init__(self):
+    if os.name == 'posix':    #Linux and Mac
+      self.RED =  '\033[91m'
+      self.GREEN =  '\033[92m'
+      self.ORANGE = '\033[93m'
+      self.RESET =  '\033[0m'
+      self.BOLD =  '\033[1m'
+    else: 
+      self.RED    = ""
+      self.GREEN  = ""
+      self.ORANGE = ""
+      self.RESET  = ""
+      self.BOLD   = ""
+
+
+  # pylint: disable=invalid-name
+  def __call__(self, test_record):
+    indent2 = ''.join(('\n', ' '*2))
+    indent4 = ''.join(('\n', ' '*4))
+  
+    #as_dict = util.ConvertToBaseTypes(test_record)
+    result = test_record.outcome
+    status = ''
+    color = None
+    if result == tr.Outcome.PASS:
+      color = self.GREEN
+      status = 'PASS'
+    elif result == tr.Outcome.FAIL:
+      color = self.RED
+      status = 'FAIL'
+    else:
+      color = self.ORANGE
+      status = 'ERROR'
+
+    output_lines = []
+    msg = ''.join((os.path.basename(sys.argv[0]), ': ', status))
+    msg = ''.join((color, self.BOLD, msg, self.RESET))
```

Comment:
I like building up of the output lines for a join at the end, but individual lines can also be formulated using `%` formatting.